ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/clock-timezone.j2

```diff
@@ -0,0 +1,17 @@
+{# Clock Timezone #}
+{% if clock is arista.avd.defined %}
```

Comment:
This template is should probably be named `clock.j2` instead of clock-timezone, so it can cover future keys under `clock`. Same for other templates and molecule.

Reply:
adjusted. took the initial naming from the already existing config template